in_toto/verifylib.py

```diff
@@ -167,7 +167,7 @@ def load_links_for_layout(layout, link_dir_path):
   return steps_metadata
 
 
-def run_all_inspections(layout):
+def run_all_inspections(layout, persist_inspection_links=True):
```

Comment:
Minor nit: I think it's okay to make this a non-optional argument here. 

Although not identified as such by leading underscore(s), `run_all_inspections`  is really more of an internal function that's only called by `in_toto_verify`, which passes `persist_inspection_links` no matter what.